test/integration/patients/list/patients-all.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'underscore';
+
+context('patient sidebar', function() {
+  specify('display patient data', function() {
+    cy
+      .server()
+      .routePatient((fx) => {
```

Comment:
you shouldn't need to set the data here.  It's not really accomplishing anything